.travis.yml

```diff
@@ -1,5 +1,6 @@
 sudo: false
 language: java
+dist: precise
```

Comment:
I didn't try but maybe it is possible to force `precise` for java7 only: 
```
     - jdk: oraclejdk7
       dist: precise
     - jdk: openjdk7
       dist: precise
```

Reply:
Tried that. It still failed after 8 mins. Again no reason why it fails on JDK8. Please see [here](https://travis-ci.org/cbeust/testng/jobs/271425668)

Reply:
@juherr - I think that did the trick. The JDK8 build failed due to test failures. Guess there was some momentary glitch in retrieving the logs. Can you please help get this merged ?